vendor/bat-native-ads/include/bat/ads/mojom.h

```diff
@@ -13,6 +13,9 @@ namespace ads {
 
 using Environment = mojom::BraveAdsEnvironment;
 
+using SysInfo = mojom::BraveAdsSysInfo;

```

Comment:
Why are you doing this? The original ones were just here to map pre-mojom class names. We should be using the full mojo class names and shouldn't map SysInfo <-> BraveAdsSysInfo. That just makes things harder to read and understand

Reply:
Legacy from Rewards, however I believe this needs revisiting in its own PR to resolve all mojo.h aliases